content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Introducing the Istio Operator
+description: Introduction to Istio's new operator-based installation and control plane management feature.
+publishdate: 2019-11-04
+subtitle:
+attribution: Martin Ostrowski (Google), Frank Budinsky (IBM)
+keywords: [install,configuration,istioctl,operator]
+target_release: 1.4
+---
+
+Kubernetes [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) provide
+a pattern for encoding human operations knowledge in software and are a popular way to simplify
+the administration of software infrastructure components.
+Because Istio is complex to administer, it's a natural candidate for an operator.
+
+Up until now, [Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio.
+In addition to needing to install another tool, this approach is not so desireable because it lacks
+schematic validation, allowing invalid configurations to be applied, which can then only be detected
+after installation. With operators, on the other hand, the API is schematized in a
+CRD (Custom Resource Definition) which is validated.
+
+The newly released Istio operator simplifies the common administrative tasks of installation,
+upgrade, and complex configuration changes for Istio.
+Starting with Istio 1.4, the [Helm installation](/docs/setup/install/helm/) approach has been deprecated
+in favor of a new operator-based [installation using {{< istioctl >}}](/docs/setup/install/operator/).
+Upgrading from Istio 1.4 onward (that is, versions not initially installed with Helm)
+will also be done using a new [{{< istioctl >}} upgrade feature](/docs/setup/upgrade/istioctl-upgrade/).
+
+## Istio Control Plane API
+
+An operator implementation requires a CRD (Custom Resource Definition) to define the API.
+The Istio operator CRD is defined by the [`IstioControlPlane` API](/docs/reference/config/istio.operator.v1alpha12.pb/).
+This API supports all of Istio's current [configuration profiles](/docs/setup/additional-setup/config-profiles/)
+using a single field to select the profile. For example, the following CRD configures Istio using the `demo` profile:
```

Comment:
I think here you are referring to CR custom resource and not CRD which is just defines a structure of a custom resource. The instantiation of CRD is CR, which is confirmed by the output below.

Reply:
Thanks, good point although, unlike CRD, I don't think CR is a recognized acronym, so I'll just reword it to `the following IstioControlPlane resource configures Istio ...`